### PR TITLE
Audit Information Changes

### PR DESCRIPTION
**Pull request by jperson1**
# Audit Information Changes - Schema, Tests, and HTML Cleanup

A whole bunch of stuff. 

Issue #1278. 

## Changes:
1. A new schema for the audit information form, and their related tests. Now, it validates against that schema on upload. And, it actually uploads somewhere! 
3. A simple set of Cypress tests for the audit info page. Basically, "Can I fill out the form and does it submit?". 
4. HTML cleanup for the audit info page. Now with 50% more for loops! 
5. Audit info form will populate with the data that currently exists in the database. If you fill it out once, you don't have to do it again!

### CODE REVIEW

**Comment by jperson1** [comment]
Posting as a draft to move things along (and so it doesn't get lost). 

Tagged @jadudm and @tadhg-ohiggins as they've contributed to the branch.

File: backend/audit/get_agency_names.py [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
+                _agency_names[row[0]] = row[1]
+    # Now, create an OrderedDict of all the values
+    agency_names = OrderedDict(sorted(_agency_names.items(), key=lambda tupl: tupl[0]))
+
     return agency_names
+
+
+def get_gaap_results():
+    sonnet = "./schemas/source/base/GAAP.libsonnet"
+    json_str = _jsonnet.evaluate_snippet(sonnet, open(sonnet).read())
```
Comment (tadhg-ohiggins) [inline backend/audit/get_agency_names.py:36]:
```suggestion
    json_str = _jsonnet.evaluate_file(sonnet)
```

File: backend/audit/test_validators.py [hunk 2]
```diff
@@ -78,30 +78,32 @@ def test_validation_is_applied(self):
...
-        simple["FederalAwards"]["federal_awards"][0]["program"][
-            "three_digit_extension"
-        ] = f"{extension}"
-
-        with self.assertRaises(
-            ValidationError,
-            msg=f"ValidationError not raised with prefix = {prefix}, extension = {extension}",
-        ):
-            validate_federal_award_json(simple)
+    # 2023-07-31 commenting out; see discussion on
```
Comment (tadhg-ohiggins) [inline backend/audit/test_validators.py:81]: @jperson1 I think we're not going to do to this in future, could you delete this entire commented-out block?

File: backend/audit/views.py [hunk 3]
```diff
@@ -515,20 +545,30 @@ def post(self, request, *args, **kwargs):
...
         try:
             sac = SingleAuditChecklist.objects.get(report_id=report_id)
-            form = AuditInfoForm(request.POST, request.FILES)
+            form = AuditInfoForm(request.POST)
 
             if form.is_valid():
-                # Save the form, yay!
-                logger.info("Audit info form passed validation.")
+                form.clean_booleans()
+                print("look at me", form.cleaned_data)
```
Comment (tadhg-ohiggins) [inline backend/audit/views.py:552]:
```suggestion
                
```

**Review by jadudm — APPROVED** [review]
Thank you for the changes. Looks good!

**Pull request merged**